cli/command/formatter/stats.go

```diff
@@ -145,8 +145,10 @@ func (c *containerStatsContext) Container() string {
 
 func (c *containerStatsContext) Name() string {
 	c.AddHeader(nameHeader)
-	name := c.s.Name[1:]
-	return name
+	if len(c.s.Name) > 1 {
+		return c.s.Name[1:]
+	}
+	return ""
```

Comment:
I wonder this should be `"--"` https://github.com/docker/docker/pull/30281/files#diff-02748098eadf6b07cde6c7d8f8bbc9c7R149

cc @yongtang